apps/base-docs/docs/building-with-base/guides/account-abstraction/overview.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Overview
+description: What's in this learning material.
```

Comment:
Needs `description`

Reply:
That is the title and description!  Though open to ideas if you have better for the Overview doc.